libraries/AP_Vehicle/AP_Vehicle.h

```diff
@@ -56,6 +56,8 @@ class AP_Vehicle : public AP_HAL::HAL::Callbacks {
 
     static AP_Vehicle *get_singleton();
 
+    void setup(void) override final;
```

Comment:
let's add a comment above the setup function to explain what it's for

Reply:
Done.  I've also moved it in the .cpp file to be in an equivalent position as in the header.